subprojects/tooling-api/src/testFixtures/groovy/org/gradle/integtests/tooling/fixture/ToolingApi.groovy

```diff
@@ -186,7 +186,15 @@ class ToolingApi implements TestRule {
             connector.useInstallation(dist.gradleHomeDir.absoluteFile)
         }
         connector.embedded(embedded)
-        connector.searchUpwards(false)
+        if (GradleVersion.version(dist.getVersion().version) < GradleVersion.version("6.0")) {
```

Comment:
❓ Shouldn't this be 7.0?

Reply:
I went with 6 because that's the moment when we deprecated the behaviour. And the solution with the settings file is more correct IMO (Gradle 6+ builds should always have a settings file).

Reply:
Makes sense. Thanks for the explanation.